drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -193,7 +193,8 @@ def set_hud_project_id_from_project_pk_unless_wip
     joins(:client).merge(Hmis::Hud::Client.age_group(start_age: start_age, end_age: end_age))
   end
 
-  scope :exited, -> { left_outer_joins(:exit).where(ex_t[:ExitDate].not_eq(nil)) }
+  scope :exited, -> { joins(:exit).where(ex_t[:ExitDate].not_eq(nil)) }
+  scope :auto_exited, -> { joins(:exit).where(ex_t[:ExitDate].not_eq(nil).and(ex_t[:auto_exited].not_eq(nil))) }
```

Comment:
This is fine as is, but could be written with a merge to allow for future changes to the exit `auto_exited` scope.
```suggestion
  scope :auto_exited, -> { joins(:exit).merge(Hmis::Hud::Exit.auto_exited.where.not(exit_date: nil) }
```